src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCount.java

```diff
@@ -53,12 +72,17 @@ public double toAltAlleleFraction() {
     }
 
     /**
-     * Returns the minor-allele fraction.
-     * @return      alternate-allele fraction
+     * Returns the maximum likelihood estimaate of the minor-allele fraction.
+     *
+     * The likelihood of a alt reads and r ref reads is proportional to f^a(1-f)^r + f^r(1-f)^a, where f in [0,1/2] is
+     * the minor allele fraction and NOT the alt allele fraction.  The two terms derive from the a priori equally likely
+     * cases that the alt and ref alleles are the minor allele, respectively.  This likelihood is unimodal on [0,1/2]
+     * so numerical max-finding is straightforward.
+     *
+     * @return      minor allele fraction
      */
     public double toMinorAlleleFraction() {
```

Comment:
this method name might be a little confusing.  maybe "estimateMinorAlleleFraction" instead (although this is a little long, and certainly "estimateAlternateAlleleFraction" is also unwieldy...).  or maybe just clarify in the doc that this does NOT give you the original formula and change the return tag to "maximum-likelihood estimate of the minor-allele fraction" to be absolutely clear.